runtime/runtime/src/verifier.rs

```diff
@@ -72,12 +75,9 @@ pub fn check_storage_stake(
     } else {
         // Check if the account is a zero balance account. The check is delayed until here because
         // it requires storage reads
-        if checked_feature!(
-            "protocol_feature_zero_balance_account",
-            ZeroBalanceAccount,
-            current_protocol_version
-        ) && is_zero_balance_account(account_id, account, state_update)
-            .map_err(|e| StorageStakingError::StorageError(e.to_string()))?
+        if checked_feature!("stable", ZeroBalanceAccount, current_protocol_version)
+            && is_zero_balance_account(account_id, account, state_update)
```

Comment:
This call right here introduces a lot of uncharged IO after every receipt.

Best case:
- account has code deployed => no extra IO

Medium case:
- account has no code but it has state => 1 extra trie key lookup (lets say 10 DB reads assuming a moderate depth of 10 trie nodes, or just 1 DB read with flat storage)

Worst case:
- account has no contract and no code but it has 5 full access keys and 2 function call access keys
   => up to 7 trie key and value lookups (these are close together in the trie, so let's be generous and say 10 trie nodes for the first key + 2 nodes for each of the other 6 keys + 1 DB read to get the value of each key, so this makes 1\*10 + 6\*2 + 7\*1 = 29 DB reads. With flat storage it would be reduced to 14 DB reads)

Note that if you really try, you can get much worst trie shapes that will use far more than 29 DB reads, this is what I would expect to happen naturally for any account that happens to have enough keys but no contract deployed.